package.json

```diff
@@ -70,7 +70,7 @@
     "husky": "^7.0.0",
     "inquirer": "^8.1.2",
     "js-yaml": "^4.1.0",
-    "lint-staged": "^12.0.2",
+    "lint-staged": "^11.2.6",
```

Comment:
had to downgrade this as v12 is broken

Reply:
I've read that from 12, we don't need to `git add` anymore on the staged task. have you tried this?

Reply:
This is downgrading quite a few dependencies. Likely will introduce vulnerabilities.

Reply:
This was upgraded & released very recently (few days ago), but the issue only appeared once we made changes (as lint-staged only throws errors on modified code). basically the error message is not relevant to the code, so you don't know what the lint issue is with v12 - I had a chat with @olliecurtis about this - it's to do with how we use husky to call lint-staged - in v12 it is no longer possible to do this. we'll have the same issue on all other projects